kornia/filters/kernels.py

```diff
@@ -698,34 +698,52 @@ def get_motion_kernel3d(kernel_size: int, angle: Union[torch.Tensor, Tuple[float
         torch.Tensor: the motion blur kernel.
 
     Shape:
-        - Output: :math:`(ksize, ksize)`
+        - Output: :math:`(B, kernel_size, kernel_size, kernel_size)`
```

Comment:
For next PRs. The convention I try to follow for functionals is to add the input/output shapes in the arts or return itself. Faster to read.